docs/index.md

```diff
@@ -1,14 +1,29 @@
+<div class="top">
+    <img src="/assets/sd_icon.png">
+    <h1>Welcome to Screwdriver!</h1>
+    <p>We've split documentation into 3 distinct sections:</p>
+</div>
 
-# Welcome to Screwdriver!
+<div class="row">
+    <div class="col-xs-6 col-md-4 about">
+        <h2>About</h2>
```

Comment:
should the header topics match the order of the topics in the homepage? or the other way around?